### PR TITLE
fix: enable support for account and org level urls

### PR DESCRIPTION
**Pull request by cjlee01**
## Hey, I just made a Pull Request!

enable support for account and org level urls for harness code

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by benjdlambert** [comment]
@OrkoHunter can I get a review here, or tag someone that can review this? :pray:

**Review by OrkoHunter — APPROVED** [review]
This looks good to me! cc @sandeepa-kv-jois @jenil-harness

**Comment by cjlee01** [comment]
@freben

**Comment by cjlee01** [comment]
@OrkoHunter  @camilaibs  @freben  @Rugvip any reason why checks are failing on this pr?

**Comment by OrkoHunter** [comment]
everything looks good now

**Comment by cjlee1** [comment]
@camilaibs @freben @Rugvip can i get reviews on this

**Comment by cjlee01** [comment]
@camilaibs @freben @Rugvip @benjdlambert  can i get reviews on this

**Comment by OrkoHunter** [comment]
@backstage/maintainers Please merge this! 🙏

File: .changeset/witty-bears-behave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration': patch
+---
+
+Updated functions for getHarnessEditContentsUrl, getHarnessFileContentsUrl, getHarnessArchiveUrl, getHarnessLatestCommitUrl and parseHarnessUrl to handle account and org level urls
```
Comment (benjdlambert) [inline .changeset/witty-bears-behave.md:5]:
```suggestion
Updated functions for `getHarnessEditContentsUrl`, `getHarnessFileContentsUrl`, `getHarnessArchiveUrl`, `getHarnessLatestCommitUrl` and `parseHarnessUrl` to handle account and org level urls
```

**Review by benjdlambert — CHANGES_REQUESTED** [review]
Sorry, summer vacations are in full swing so this has been waiting for people to come back from them! Just a comment about the changeset, you could even remove the specificity a little bit and just simplify it overall instead.

**Comment by cjlee01** [comment]
@benjdlambert fixed comment

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**